user/troubleshooting/gui-troubleshooting.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: doc
+title: GUI Troubleshooting
+permalink: /doc/gui-troubleshooting/
+---
+
+# GUI Troubleshooting
+
+## Can't click on anything after connecting 4k external display ##
+
+When you connect a 4K external display, you may be unable to click on anything but a small area in the upper-right corner.
+
+When a qube starts, a fixed amount of RAM is allocated to the graphics buffer called video RAM.
+This buffer needs to be at least as big as the whole desktop, accounting for all displays that are or will be connected to the machine.
+By default, it is as much as needed for the current display and an additional full HD (FHD) display (1920×1080 8 bit/channel RGBA).
+This logic fails when the machine has primary display in FHD resolution and, after starting some qubes, a 4K display is connected.
+If the buffer is too small, and internal desktop resize fails.
+
+The solution to this problem is to increase the minimum size of the video RAM buffer, as explained in [GUI Configuration](/doc/gui-configuration/#video-ram-adjustment-for-high-resolution-displays). 
+
+
+## Screen blanks / Weird computer glitches like turning partially black or black boxes ##
+
+You may encountering seemingly random screen blanking while using Qubes, where the screen will black and shows the logon screen, yet, only the active window will show when you move the mouse or use the keyboard. Sometimes, you will get random black screens or black boxes. 
+
+Similarly, while working, the XScreenSaver dialog may pop up (indicating the screen is locked) and the screen goes black. However, the screen is not locked, and you have to move a window to redraw the screen. 
+
+If you are experiencing the any of the above symptoms, try disabling the window compositor:
+
+    Q → System Tools → Window Manager Tweaks → Compositor → uncheck “Enable display compositing”
+
+## Post installation, screen goes black and freezes following LUKS decryption ##
+
+After installing Qubes, you may experience a black screen after entering your LUKS decryption password. 
+To fix the problem, use your preferred text editor (`nano` works) to edit `/mnt/sysimage/boot/efi/EFI/qubes/xen.cfg`, adding the `efi=no-rs` option to the end of the `options= line`. For example:
```

Comment:
Note this path applies when running from the installer (either directly after installation, before the reboot, or by starting the installer again in recovery mode). On the actual installed system, it is `/boot/efi/EFI/qubes/xen.cfg` - but it may be hard to access directly when your system won't boot.